### PR TITLE
Installer Metrics update: ns default and bugfix

### PR DESCRIPTION
**Pull request by jmckulk**
This change adds in a default namespace for the metrics deployment. The default
will now be pgo if you use the installer image. The default namespace is also
updated in the inventory and ansible defaults for pgo-metrics.

This change also makes a bugfix where the fsGroup wasn't be set on the grafana
and prometheus deployments. This caused a permission error when the processes
tried to write to a required directory.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?
tested in gke with and without the installer image. Also tested installer image in OpenShift (GCE)

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

**What is the new behavior (if this is a feature change)?**

**Other information**:
[ch7789]
[ch7844]

### CODE REVIEW

File: docs/content/Installation/postgres-operator-installer/kubectl-install.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ The deploy job can be used to perform different deployment actions for the
 PostgreSQL Operator. If you run the job it will install the operator but you can
 change the deployment action by updating the `DEPLOY_ACTION` environment
 variable in the `deploy.yml` file. This variable can be set to `install`,
-`update`, and `uninstall`. Each time a job is run you will need to cleanup the
-job using the command below.
+`update`, `uninstall`, `install-metrics`, and `uninstall-metrics`. Each time a
+job is run you will need to cleanup the job using the command below.
```
Comment (jkatz) [inline docs/content/Installation/postgres-operator-installer/kubectl-install.md:96]: Which command below? Can you reference a section?

File: installers/ansible/roles/pgo-metrics/templates/grafana-deployment.json.j2 [hunk 2]
```diff
@@ -22,7 +22,10 @@
             "spec": {
                 "securityContext": {
 {% if grafana_supplemental_groups is defined and grafana_supplemental_groups != '' %}
+                    "fsGroup": 26,
                     "supplementalGroups": [{{ grafana_supplemental_groups }}]
+{% else %}}
+                    "fsGroup": 26
 {% endif %}
```
Comment (jkatz) [inline installers/ansible/roles/pgo-metrics/templates/grafana-deployment.json.j2:29]: I think you may want:

```json
"securityContext": {
    {% if grafana_supplemental_groups is defined and grafana_supplemental_groups != '' %}
        "supplementalGroups": [{{ grafana_supplemental_groups }}],
    {% endif %}
    "fsGroup": 26
}
```

File: installers/ansible/roles/pgo-metrics/templates/prometheus-deployment.json.j2 [hunk 3]
```diff
@@ -21,8 +21,11 @@
...
             "spec": {
                 "securityContext": {
-{% if prometheus_supplemental_groups is defined and prometheus_supplemental_groups != '' %}
-                    "supplementalGroups": [{{ prometheus_supplemental_groups }}]
+{% if grafana_supplemental_groups is defined and grafana_supplemental_groups != '' %}
+                    "fsGroup": 26,
+                    "supplementalGroups": [{{ grafana_supplemental_groups }}]
+{% else %}}
+                    "fsGroup": 26
 {% endif %}
```
Comment (jkatz) [inline installers/ansible/roles/pgo-metrics/templates/prometheus-deployment.json.j2:29]: Similar to comment in the grafana-deployment file. Also, I believe you want `prometheus_supplemental_groups` here.

File: docs/content/Installation/postgres-operator-installer/kubectl-install.md (hunk 1 above)
Comment (jmckulk) [reply docs/content/Installation/postgres-operator-installer/kubectl-install.md:96]: this file is removed in other pr. Can i make the change there?

Comment (jkatz) [reply docs/content/Installation/postgres-operator-installer/kubectl-install.md:96]: Yes

File: installers/ansible/roles/pgo-metrics/templates/prometheus-deployment.json.j2 [hunk 4]
```diff
@@ -21,9 +21,10 @@
             },
             "spec": {
                 "securityContext": {
-{% if prometheus_supplemental_groups is defined and prometheus_supplemental_groups != '' %}
-                    "supplementalGroups": [{{ prometheus_supplemental_groups }}]
+{% if grafana_supplemental_groups is defined and grafana_supplemental_groups != '' %}
+                    "supplementalGroups": [{{ grafana_supplemental_groups }}],
```
Comment (jkatz) [inline installers/ansible/roles/pgo-metrics/templates/prometheus-deployment.json.j2:25]: This needs to be `prometheus_supplemental_groups`

**Pull request merged**